bitnami/kibana/Chart.yaml

```diff
@@ -25,4 +25,4 @@ name: kibana
 sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/kibana
   - https://www.elastic.co/products/kibana
-version: 10.2.6
+version: 10.2.7
```

Comment:
Version `10.2.7` has been already released, could you please bump a new version?
https://github.com/bitnami/charts/blob/main/bitnami/kibana/Chart.yaml